requirements.txt

```diff
@@ -0,0 +1,13 @@
+cycler==0.10.0
+joblib==0.14.0
+kiwisolver==1.1.0
+matplotlib==3.0.3
+numpy==1.17.3
+pandas==0.25.2
+pyparsing==2.4.2
+python-dateutil==2.8.0
+pytz==2019.3
+scikit-learn==0.21.3
+scipy==1.3.1
+six==1.12.0
+sklearn==0.0
```

Comment:
Version 0.0?

Reply:
My bad. Did not notice shift from ```pip install sklearn``` to ```pip install scikit-learn```
Reference [link](https://pypi.org/project/sklearn/)

Reply:
Yeah